google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClientFactory.java

```diff
@@ -189,8 +189,9 @@ public BigtableDataClient createForInstance(
   }
 
   // Update stub settings to use shared resources in this factory
-  private void patchStubSettings(StubSettings.Builder stubSettings) {
+  private void patchStubSettings(EnhancedBigtableStubSettings.Builder stubSettings) {
     stubSettings
+        .setRefreshingChannel(false)
```

Comment:
Please add comment, something along the lines of:

Channel refreshing only needs to be configured on the shared ClientContext, derivative clients don't have the ability to reconfigure the refresh logic

Reply:
Added comments.